runtime/ptrace_ios.h

```diff
@@ -0,0 +1,14 @@
+#ifndef FLUTTER_RUNTIME_PTRACE_IOS_H_
```

Comment:
License block. You can copy the same from a sibling file.

Reply:
Done.